.circleci/config.yml

```diff
@@ -32,6 +32,9 @@ jobs:
             source pyodide-env/bin/activate
             pip install pytest pytest-xdist pytest-instafail selenium PyYAML flake8
 
+            # Download BLAS/LAPACK 
+            git clone https://github.com/adrianbg/CLAPACK-WA.git packages/scipy/CLAPACK-WA
```

Comment:
Eventually, this should probably happen as part of the Scipy package build, or maybe its own package that Scipy is dependent on.